app/views/pages/_search.html.erb

```diff
@@ -1,18 +1,8 @@
-<div class="grid-row">
-  <div class="column-two-thirds">
-    <h1 class="heading-xlarge">
-      <%= t('.find_gov_data') %>
-    </h1>
-    <p class="lede">
-    <%= t('.lede') %>
-    </p>
-  </div>
-</div>
-<div class="grid-row">
-  <div class="column-two-thirds dgu-search-box">
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds dgu-search-box">
     <form action="/search" method="GET">
-      <label for="q" class="visuallyhidden"><%= t('.accessibility.search_box_label') %></label>
-      <input id="q" name="q" type="text" class="form-control dgu-search-box__input" /><!-- Make sure there is no space between the input and button else the button will become detached --><button type="submit" class="dgu-search-box__button">
+      <label for="search" class="govuk-visually-hidden"><%= t('.accessibility.search_box_label') %></label>
+      <input id="search" name="search" type="text" class="form-control dgu-search-box__input" /><!-- Make sure there is no space between the input and button else the button will become detached --><button type="submit" class="dgu-search-box__button">
```

Comment:
If this comment is essential, suggest it be done as a ruby comment, otherwise it's visible in the public source (although this seems like something that could be fixed).